<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addStaff</class>
 <widget class="QMainWindow" name="addStaff">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1441</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/images/icons/logoicon.ico</normaloff>:/new/images/icons/logoicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {background-image: url(:/new/images/Images/background_gradient_blue_02_jpg_max.jpg);
}
QPushButton {
    background-color: rgb(0, 170, 127);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
	color:white;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgb(224, 0, 0);
    border-style: inset;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">qproperty-alignment: AlignCenter;
color:white;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Fill the information about new staff&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Name</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="fName">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel { color: white; }
QLineEdit { border-radius: 3px; }</string>
               </property>
               <property name="text">
                <string>First Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Sfrstname">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="mName">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel { color: white; }
QLineEdit { border-radius: 3px; }</string>
               </property>
               <property name="text">
                <string>Middle Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Smidname">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel { color: white; }
QLineEdit { border-radius: 3px; }</string>
                 </property>
                 <property name="text">
                  <string>Last Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Slastname">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="title">
             <string>Address</string>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel { color: white; }
QLineEdit { border-radius: 3px; }</string>
               </property>
               <property name="text">
                <string>Permanent Address</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="Spaddress">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_6">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel { color: white; }
QLineEdit { border-radius: 3px; }</string>
               </property>
               <property name="text">
                <string>Temporary Address</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="Staddress">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="title">
             <string>Sex</string>
            </property>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QRadioButton" name="sMale">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QRadioButton { color: white; }
QLineEdit { border-radius: 3px; }</string>
               </property>
               <property name="text">
                <string>Male</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="sFemale">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QRadioButton { color: white; }</string>
               </property>
               <property name="text">
                <string>Female</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QRadioButton" name="sOther">
               <property name="font">
                <font>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QRadioButton { color: white; }</string>
               </property>
               <property name="text">
                <string>Other</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="title">
             <string>Contact</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel { color: white; }
QLineEdit { border-radius: 3px; }</string>
                 </property>
                 <property name="text">
                  <string>    Phone.No</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Sphone">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel { color: white; }
QLineEdit { border-radius: 3px; }</string>
                 </property>
                 <property name="text">
                  <string>Email</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="Semail">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="title">
               <string>Other</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="font">
                    <font>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLabel { color: white; }
QLineEdit { border-radius: 3px; }</string>
                   </property>
                   <property name="text">
                    <string>Salary</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="Ssalary">
                   <property name="font">
                    <font>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="font">
                    <font>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLabel { color: white; }
QLineEdit { border-radius: 3px; }</string>
                   </property>
                   <property name="text">
                    <string>Position</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="Sposition">
                   <property name="font">
                    <font>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_6">
              <property name="styleSheet">
               <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
              </property>
              <property name="title">
               <string>Descriptions</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QTextEdit" name="StaffDescription"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_7">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="title">
             <string>Qualifications</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLineEdit" name="Qualification1"/>
             </item>
             <item>
              <widget class="QLineEdit" name="Qualification2"/>
             </item>
             <item>
              <widget class="QLineEdit" name="Qualification3"/>
             </item>
             <item>
              <widget class="QLineEdit" name="Qualification4"/>
             </item>
             <item>
              <widget class="QLineEdit" name="Qualification5"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="pic">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="resources.qrc">:/img/na.jpg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="chooseImage">
            <property name="text">
             <string>Choose Image</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="addmore">
          <property name="text">
           <string>Add More Records</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="sFormSubmit">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="back">
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
